apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -69,24 +72,78 @@ export class ApplicationController {
 
     return application
   }
+
   // TODO REMOVE
   @Get()
-  @ApiOkResponse({ type: Application, isArray: true })
-  async findAll(@Query('typeId') typeId: string): Promise<Application[]> {
+  @ApiOkResponse({ type: ApplicationResponseDto, isArray: true })
+  @UseInterceptors(ApplicationSerializer)
+  async findAll(
+    @Query('typeId') typeId: string,
+  ): Promise<ApplicationResponseDto[]> {
     if (typeId) {
       return this.applicationService.findAllByType(typeId as ApplicationTypes)
     } else {
       return this.applicationService.findAll()
     }
   }
 
-  @Post()
-  @ApiCreatedResponse({ type: Application })
+  @Get('applicants/:nationalRegistryId/applications')
```

Comment:
you should add tests for these new endpoints in the e2e spec

Reply:
will do